index.html

```diff
@@ -7935,7 +7935,8 @@ <h2 id="MediaStreamAudioSourceNode">
           connections to the input are up-mixed/down-mixed to the number of
           channels of the input.
         </p>
-        <dl title="interface MediaStreamAudioDestinationNode : AudioNode"
+        <dl title=
+        "[Constructor(AudioContext context, optional AudioNodeChannelOptions options)]interface MediaStreamAudioDestinationNode : AudioNode"
```

Comment:
This appears to be lacking a property bag in which to provide the initial `MediaStream` destination, following the handling of media stream sources in #897 .